File: 2st/Yeonwoo/날짜 비교하기.py
def solution(date1, date2):
    answer = 0
    if date1[0] < date2[0]:
        answer = 1
    elif date1[0] == date2[0] and date1[1] < date2[1]:
        answer = 1
    elif date1[0] == date2[0] and date1[1] == date2[1] and date1[2] < date2[2]:
        answer = 1
    return answer

File: 2st/Yeonwoo/날짜 비교하기.java
class Solution {
    public int solution(int[] date1, int[] date2) {
        if (date1[0] < date2[0]) {
            return 1;
        } else if (date1[0] > date2[0]) {
            return 0;
        }else {
            if (date1[1] < date2[1]) {
                return 1;
            } else if (date1[1] > date2[1]) {
                return 0;
            } else {
                if (date1[2] < date2[2]) {
                    return 1;
                } else {
                    return 0;
                }
            }
        }
    }
}